src/psl/psl-node.c

```diff
@@ -209,6 +222,25 @@ void psl_set_clock(psl_node_t p, psl_node_t clk)
    object_write_barrier(&(p->object), &(clk->object));
 }
 
+tree_t psl_message(psl_node_t p)
+{
+   item_t *item = lookup_item(&psl_object, p, I_MESSAGE);
+   assert(item->object != NULL);
+   return container_of(item->object, struct _tree, object);
+}
+
+bool psl_has_message(psl_node_t p)
+{
+   return lookup_item(&psl_object, p, I_MESSAGE)->object != NULL;
+}
+
+void psl_set_message(psl_node_t p, tree_t m)
+{
+   tree_assert_expr(m);
```

Comment:
I think we can just drop this `tree_assert_expr` here. I added them very early when I started NVC but I don't think it finds any useful bugs.